extensions/vertx-core/runtime/src/main/java/io/quarkus/vertx/core/runtime/SSLConfigHelper.java

```diff
@@ -0,0 +1,119 @@
+package io.quarkus.vertx.core.runtime;
+
+import java.util.regex.Pattern;
+
+import io.quarkus.vertx.core.runtime.config.JksConfiguration;
+import io.quarkus.vertx.core.runtime.config.PemKeyCertConfiguration;
+import io.quarkus.vertx.core.runtime.config.PemTrustCertConfiguration;
+import io.quarkus.vertx.core.runtime.config.PfxConfiguration;
+import io.vertx.core.net.*;
+
+public class SSLConfigHelper {
+
+    private static final Pattern COMMA_PATTERN = Pattern.compile(",");
```

Comment:
This should be done directly in the config class. List are comma-separated by default. See the last example from https://github.com/eclipse/microprofile-config/blob/master/spec/src/main/asciidoc/configexamples.asciidoc.

Reply:
Thanks for pointing this out, I will update the config object.